app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -0,0 +1,8 @@
+<p>Hello <%= @resource.email %>!</p>
```

Comment:
Is the feature to send mails functional?

Reply:
This file was added automatically using the devise gem, at the moment it is not functional, I will remove this file from the project